frame.test.js

```diff
@@ -0,0 +1,55 @@
+const Frame = require('./Frame');
+
```

Comment:
const Frame = require('./Frame');       should be the following instead please to match the name of the file you are testing against (lower case f for frame): const Frame = require('./frame');         Thank you. Your code looks really good.